### PR TITLE
Makes sabre sheathe use ItemSlots

### PR DESCRIPTION
**Pull request by joshepvodka**
## About the PR

Makes the captain's sabre sheath use an item slot instead of opening a container.

## Why / Balance

Because the sheath already had a single slot for the sabre.

## Technical details

Would be cool if someone got sound effects for it. (not me though)
## Media

https://github.com/space-wizards/space-station-14/assets/86210200/1477d352-e4d0-4e2a-b53d-958834183a3c

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
:cl:
- tweak: The captain's sabre sheath now uses a slot instead of a container UI.

### CODE REVIEW

**Comment by deltanedas** [comment]
troller this still needs ctrl+e fixing

**Comment by joshepvodka** [comment]
fixing what?

**Comment by deltanedas** [comment]
ctrl+e doesnt work

**Comment by Emisse** [comment]
id say thats out of scope

**Comment by deltanedas** [comment]
ok then just change the filled sheath from StorageFill to ContainerFill

**Comment by Emisse** [comment]
mirror is rewriting the system

**Comment by mirrorcult** [comment]
> mirror is rewriting the system

#22815

File: Resources/Prototypes/Entities/Clothing/Belt/belts.yml [hunk 1]
```diff
@@ -409,12 +409,17 @@
...
-      tags:
-        - CaptainSabre
+  - type: Item
+    size: Ginormous
+  - type: ItemSlots
+    slots:
+      item:
+        name: Sabre
+        insertVerbText: Sheathe
+        ejectVerbText: Unsheathe
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Clothing/Belt/belts.yml:419]: localize this with .ftl files.

https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html?highlight=local#localization

**Review by EmoGarbage404 — APPROVED** [review]

File: Resources/Prototypes/Entities/Clothing/Belt/belts.yml (hunk 1 above)
Comment (joshepvodka) [reply Resources/Prototypes/Entities/Clothing/Belt/belts.yml]: Done.

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**